gobblin-wmf-core/pom.xml

```diff
@@ -78,5 +78,10 @@
             <artifactId>testng</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Good catch on the missing direct dependency. Looks like we have a few more (`./mvnw dependency:analyze`). The mockito version is missing and needs to be defined in the parent pom.

Reply:
addressed in another commit